apps/srt-file-transmit.cpp

```diff
@@ -25,12 +25,23 @@ bool Download(UriParser& srt, UriParser& file);
 const logging::LogFA SRT_LOGFA_APP = 10;
 
 size_t g_buffer_size = 1456;
+bool g_skip_flushing = false;
```

Comment:
If these globals are only used in this file, please make  them 'static'

Reply:
This file is an exclusive ingredient for an executable file (of the same rootname), which does not share its symbols with any other parts of any, even potential, software. Every global variable defined here will be inaccessible elsewhere, no matter if it's static or not.

Reply:
The global is accessible by other components used by this file. At any rate this is just good practice and good practices do matter so comment was automatic and mostly out of courtesy as I assumed non-static global was not intentional.

Reply:
There is no possible way how they can be accessible the way you described. Please make a trial code snippet that could prove it.

Reply:
No need for additional code snippets. Current use of true_names/false_names globals is good example of how it can be done and of what not to do.

Reply:
The false_names/true_names variables have nothing to do with the currently commented code. These variables here are not accessible anywhere outside this file by any imaginable method in the current form of declaration, whether they are static or not.